pkg/parse/run_test.go

```diff
@@ -426,42 +432,43 @@ func TestRun(t *testing.T) {
 	}
 }
 
-func TestBackoffRetryCount(t *testing.T) {
-	parser := newParser(t, FileSource{}, false, 10*time.Microsecond, 150*time.Microsecond)
-	testState := &namespacecontroller.State{}
-	reimportCount := 0
-	retryCount := 0
-	testIsDone := func(reimportCount *int, _ *int) bool {
-		return *reimportCount == 35
-	}
+// func TestRunRetryBackoffMaxAttempts(t *testing.T) {
```

Comment:
Need to replace this with an actual test of the TimedEventProducer, since that's where the backoff logic is now.

Reply:
Added backoff tests to TestFunnel_Start